src/wallet/scriptpubkeyman.h

```diff
@@ -123,20 +123,14 @@ class CKeyPool
     template<typename Stream>
     void Serialize(Stream& s) const
     {
-        int nVersion = s.GetVersion();
-        if (!(s.GetType() & SER_GETHASH)) {
-            s << nVersion;
-        }
+        s << int{259900}; // Unused field, writes the highest client version ever written
```

Comment:
nit: in the comment does "highest client version ever written" mean that this is the highest client version that can be written or that we've ever seen written?

Suggestion
"Unused field, writes the highest possible client version"

feel free to ignore this nit aswell!